pkg/volume/local/local.go

```diff
@@ -16,6 +16,11 @@ limitations under the License.
 
 package local
 
+//
```

Comment:
I would suggest move this comment to where the lock is used below. Nobody will check notes here I'm afraid.

Reply:
Good suggestion! Done.